<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start food:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="food">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="money">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start money:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="item">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start item:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="area_size">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start area:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upgrade_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Upgrade</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Starter changes:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="daily_money">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>item</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>money</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>180</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Daily changes:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="daily_food">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="daily_item">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="daily_upgrade_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Upgrade</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>food</string>
    </property>
   </widget>
   <widget class="QPushButton" name="map_buton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Map</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Market ücreti:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="shop_salary">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mağaza ücreti:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="item_salary">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="emlak_salary">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>emlak ücreti</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maaş:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>yiyecek ücreti</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="item_price">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fiyatlar:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="food_price">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>eşya_ücreti</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mülkler ve sahipleri:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="ownered_by_admin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>501</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>sahip id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>soyad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>mülk id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>mülk tipi</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="market_text">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Build business cost:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Market</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Emlak</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mağaza</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emlak_text">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="supermarket_text">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Build_upgrade_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Upgrade</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="second_text">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>420</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="level_upgrade_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Upgrade</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="first_text">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="third_text">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Level_oran</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
